app/models/hardware.rb

```diff
@@ -154,4 +154,11 @@ def m_snapshots(parent, xmlNode, _deletes)
   def m_volumes(parent, xmlNode, _deletes)
     Volume.add_elements(parent, xmlNode)
   end
+
+  def sockets
+    return 0 unless logical_cpus && cores_per_socket
+    logical_cpus / cores_per_socket
```

Comment:
@imtayadeway Since https://github.com/ManageIQ/manageiq/pull/4738 was merged yesterday, you can use the new column names.  I don't think you need this method, `cpu_sockets` should be good enough, right?